pom.xml

```
@@ -36,10 +36,11 @@
 
     <!-- Update occasionally based on LTS, but don't overrun what's in Alpine:
          https://pkgs.alpinelinux.org/packages?name=nodejs-current -->
-    <node.version>14.14.0</node.version>
+    <node.version>21.6.2</node.version>
+    <pnpm.version>8.15.3</pnpm.version>
```

alpine doesn't have this version yet.
```suggestion
    <pnpm.version>8.15.1</pnpm.version>
```

 Later when we do the same in zipkin, we have to update the script accordingly https://github.com/openzipkin/zipkin/blob/master/build-bin/maybe_install_npm

---

tl;dr; is only way we should change the build tool here is if we also change it in zipkin (including any README citations) because otherwise it is harder or more annoying when context switching.